zha/zigbee/cluster_handlers/homeautomation.py

```diff
@@ -61,6 +62,30 @@ class MeasurementType(enum.IntFlag):
         POWER_QUALITY_MEASUREMENT = 256
 
     REPORT_CONFIG = (
+        AttrReportConfig(
+            attr=ElectricalMeasurement.AttributeDefs.ac_voltage_multiplier.name,
+            config=REPORT_CONFIG_IMMEDIATE,
+        ),
+        AttrReportConfig(
+            attr=ElectricalMeasurement.AttributeDefs.ac_voltage_divisor.name,
+            config=REPORT_CONFIG_IMMEDIATE,
+        ),
+        AttrReportConfig(
+            attr=ElectricalMeasurement.AttributeDefs.ac_current_multiplier.name,
+            config=REPORT_CONFIG_IMMEDIATE,
+        ),
+        AttrReportConfig(
+            attr=ElectricalMeasurement.AttributeDefs.ac_current_divisor.name,
+            config=REPORT_CONFIG_IMMEDIATE,
+        ),
+        AttrReportConfig(
+            attr=ElectricalMeasurement.AttributeDefs.ac_power_multiplier.name,
+            config=REPORT_CONFIG_IMMEDIATE,
+        ),
+        AttrReportConfig(
+            attr=ElectricalMeasurement.AttributeDefs.ac_power_divisor.name,
+            config=REPORT_CONFIG_IMMEDIATE,
+        ),
```

Comment:
`ZCL_INIT_ATTRS` with `True` will populate `cached`. `REPORT_CONFIG` will populate `uncached`. So, the attributes will be read twice. They should be removed from `ZCL_INIT_ATTRS` at least. Relevant code: https://github.com/zigpy/zha/blob/fd6cf2f570f1778a9c0933a56ebf476caafe859d/zha/zigbee/cluster_handlers/__init__.py#L458-L460

Do note that, by default, we will always read six more attributes per smart plug on network startup now.

Reply:
Good point. I'm also noticing that `power_divisor` and `power_multiplier` are read on startup but are not set up for reporting, which they support.